rllib/policy/torch_policy.py

```diff
@@ -535,8 +535,11 @@ def load_batch_into_buffer(
                 view_requirements=self.view_requirements,
             )
 
-        # 3) Load splits into the given buffer (consisting of n GPUs).
-        slices = [slice.to_device(self.devices[i]) for i, slice in enumerate(slices)]
+        if not self.config.get("_load_only_minibatch_onto_device", False):
+            # 3) Load splits into the given buffer (consisting of n GPUs).
```

Comment:
Nit: Move the comment one line up and add the explanation of why sometimes it's better to NOT move the entire batch(es) directly onto the GPU.